src/olympia/activity/tests/test_views.py

```diff
@@ -208,6 +209,150 @@ def _set_tested_url(self, pk=None, version_pk=None, addon_pk=None):
             'version_pk': version_pk or self.version.pk})
 
 
+class TestReviewNotesViewSetCreate(TestCase):
+    client_class = APITestClient
+
+    def setUp(self):
+        super(TestReviewNotesViewSetCreate, self).setUp()
+        self.addon = addon_factory(
+            guid=generate_addon_guid(), name=u'My Addôn', slug='my-addon')
+        self.version = self.addon.latest_version
+        self.url = reverse('version-reviewnotes-list', kwargs={
+            'addon_pk': self.addon.pk,
+            'version_pk': self.addon.latest_version.pk})
+
+    def _post_reply(self):
+        return self.client.post(self.url, {'comments': u'comménty McCómm€nt'})
+
+    def get_review_activity(self):
```

Comment:
Hmm, the name is confusing to me. Shouldn't this be `get_review_activities` to at least imply some kind of list? I'd rather see `get_review_activity_queryset` maybe to imply the actual return value.